Tests/js/views/dashboard/ez-dashboardblockallcontentview.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>eZ Dashboard All Content Block View tests</title>
+</head>
+<body>
+<div class="container"></div>
+<div class="outside"></div>
+
+<script type="text/x-handlebars-template" id="dashboardblockallcontentview-ez-template" >
+    <div class="ez-allcontent-block-wrapper ez-asynchronousview">
```

Comment:
as said in a previous PR, please use the simplest template possible in tests. In this template, most classes are useless from a test point of view.